docs/Dirb.md

```diff
@@ -0,0 +1,36 @@
+# How to install and use Dirb for Mac OS
+
+## Installation 🔧
+
+All you need to do is follow the commands below, and Dirb will be installed in no time!
```

Comment:
What do you think of dissecting each command describing each step being made? I think doing so will clarify what developers are actually executing in their machine.

Reply:
Sure! Will do!